pkg/placement/binding-organizer.go

```diff
@@ -38,7 +36,7 @@ func SimpleBindingOrganizer(logger klog.Logger) BindingOrganizer {
 			resourceModes:     resourceModes,
 			eventHandler:      eventHandler,
 			workloadProjector: workloadProjector,
-			perSourceCluster:  NewMapMap[logicalcluster.Name, *simpleBindingPerCluster](nil),
+			perSourceCluster:  NewMapMap[string, *simpleBindingPerCluster](nil),
```

Comment:
It would have been nice to declared

```
type SpaceName string
func (sn SpaceName) String() string {return sn}
```

to maintain a distinction from other strings in the Go type system.